gb-dbg/src/app.rs

```diff
@@ -0,0 +1,28 @@
+use egui::{CtxRef};
+use crate::memory::GBMemoryEditor;
+
+pub struct DebugApp<T> {
+    memory_editor: GBMemoryEditor<T>,
+}
+
+impl<T> DebugApp<T> {
+
+    pub fn new(memory_editor: GBMemoryEditor<T>) -> Self {
+        Self {
+            memory_editor
+        }
+    }
+
+    pub fn draw(&mut self, ctx: &CtxRef) {
+        egui::TopBottomPanel::top("top_panel").show(ctx, |ui| {
+            ui.horizontal(|ui| {
+                ui.button("Step");
+                ui.button("Run one frame");
+                ui.button("Run one second")
+            });
+        });
```

Comment:
It is possible to have these defined in their own module ? (i.e.: here It'll be `FlowControl` or something like that)

Reply:
It will be when I get around the architecture of the Debugger